mne/viz/_3d.py

```diff
@@ -2327,6 +2316,18 @@ def plot_vector_source_estimates(stc, subject=None, hemi='lh', colormap='hot',
                             f.scene.renderer.use_depth_peeling = True
         brain.scale_data_colormap(fmin=scale_pts[0], fmid=scale_pts[1],
                                   fmax=scale_pts[2], **sd_kwargs)
+    for hemi in hemis:
+        if scale_factor is None:
+            # Compute the width of the brain
+            width = np.ptp(brain.geo[hemi].coords[:, 1])
+            # Retrieve the current hemi
+            for b in brain._brain_list:
+                if b['hemi'] == hemi:
+                    found_hemi = b['brain']
+            # Configure the glyphs scale directly
+            for layer in found_hemi.data.values():
+                glyphs = layer['glyphs']
+                glyphs.glyph.glyph.scale_factor = width * 0.1
```

Comment:
This will give different scales to the left and right hemi data, which is not good. I would take the mean width across all hemis found and apply that to all of them